pkg/kv/kvserver/replica_rangefeed_test.go

```diff
@@ -781,7 +781,7 @@ func TestReplicaRangefeedErrors(t *testing.T) {
 		pErr := <-streamErrC
 		assertRangefeedRetryErr(t, pErr, kvpb.RangeFeedRetryError_REASON_REPLICA_REMOVED)
 	})
-	t.Run(kvpb.RangeFeedRetryError_REASON_RANGE_SPLIT.String(), func(t *testing.T) {
+	t.Run(kvpb.RangeFeedRetryError_REASON_MANUAL_RANGE_SPLIT.String(), func(t *testing.T) {
```

Comment:
Do we want to add a test here where we do something like add a span configuration which forces a range split and then assert we get the other error?

Reply:
I think it will be much easier to test that split queue splits return a different rangefeed error once we plumb logic up to the range feed client. Once we have that I can test that the split error was only emitted on a manual split as oppose to when a table is created, for example. This test framework only works on the dummy span `[a,z)`, and it seems like significant work to update the span configs table properly using this arbitrary span.